packages/process/src/node/terminal-process.ts

```diff
@@ -187,7 +53,7 @@ export class TerminalProcess extends Process {
         @inject(MultiRingBuffer) protected readonly ringBuffer: MultiRingBuffer,
         @inject(ILogger) @named('process') logger: ILogger
     ) {
-        super(processManager, logger, ProcessType.Terminal, TerminalProcess.resolveExecOptions(options));
```

Comment:
Could it break some existing users? Or escaping done in another place for existing clients?

Reply:
Escaping is done at the terminal's caller level. It simplifies the role of the `TerminalProcess` class, since its original purpose was simply to start arbitrary processes inside a pty, and not necessarily inside a shell.

Reply:
Depending on how people used that class, it might be breaking, but I think it is for the best.

Reply:
a record in the changelog explaining what clients should do instead?